.gitignore

```diff
@@ -2,21 +2,43 @@ test.py
 *.bz2
```

Comment:
We don't need to include the CoreNLP errors text file that is added further down in the list of commits

Reply:
My mistake, adding to gitignore